libgloss/arc/arc_timer.h

```diff
@@ -0,0 +1,12 @@
+/*********************************************************************
+Copyright  2020 Synopsys Inc.
```

Comment:
header, comments. I would suggest to rename it as `semihost_helper.h`

Reply:
done